api/api-doc/storage_service.json

```diff
@@ -104,6 +104,68 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/toppartitions/",
+         "operations":[
+            {
+               "method":"GET",
+               "summary":"Toppartitions query",
+               "type":"toppartitions_query_results",
+               "nickname":"toppartitions_generic",
+               "produces":[
+                  "application/json"
+               ],
+               "parameters":[
+                  {
+                     "name":"table_filters",
+                     "description":"Optional list of table name filters in keyspace:name format",
+                     "required":false,
+                     "allowMultiple":false,
+                     "type":"array",
+                     "items":{
+                        "type":"string"
+                     },
+                     "paramType":"query"
+                  },
+                  {
+                     "name":"keyspace_filters",
```

Comment:
then it should be just keyspace (or ks)

Reply:
but why? Many keyspaces can be provided, so a singular here seems weird

Reply:
Did you tested it? I don't think we support array of query parameters

Reply:
yes

Reply:
not only it's working, it's also used in other places than here as well

Reply:
It's a nitpick, but from looking at the code this is a comma seperated string, the API currently does not enforce it, but it's better be correct anyhow, but I'll not fight over it